[v7r1] fixes for MultiProcessor queues (#4926)
DIRACGrid/DIRAC: docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst

CEs  sub-subsection
-------------------

> This sub-subsection specify the attributes of each particular CE of the site. Must be noticed than in eachDRAC site can be more than one CE.

Reviewer: ```suggestion
This sub-subsection specifies the attributes of each particular CE of the site. For each DIRAC site there can be more than one CE.
```

| *<CE_NAME>/CEType*                             | Type of CE, can take values as LCG or CREAM                  | CEType = ARC                   |
+------------------------------------------------+------------------------------------------------------------- +--------------------------------+
| *<CE_NAME>/LocalCEType*                        | Type of 'Inner' CE, normally empty. Default = "InProcess".   | LocalCEType = Pool             |
|                                                | Possibilities: potentially all CE types, but in practice     |                                |
|                                                | the most valid would be: InProcess, Sudo, Singularity, Pool. |                                |
|                                                | Pool CE in turn uses InProcess (Default)                     |                                |
|                                                | or Sudo or Singularity. To specify, use Pool/ce_type.        | LocalCEType = Pool/Singularity |
|                                                | This option can also go at the Queue level.                  |                                |

> +------------------------------------------------+------------------------------------------------------------- +--------------------------------+

Reviewer: There is a problem with the table syntax, it is missing a `-` in many of the `----` lines.

Author: I have fixed it here.

Reviewer: You still have to push this fix, right?
